src/components/vanilla/Spinner.tsx

```diff
@@ -10,8 +10,8 @@ export default function Spinner({ show, className }: { show?: boolean; className
         }`,
         className
       )}
-      width="24"
-      height="24"
+      width={size || "24"}
+      height={size || "24"}
       viewBox="0 0 24 24"
```

Comment:
can you test different sizes? i believe that viewBox will mess up sizes different than 24

Reply:
@mad-raccoon I tested this and it works fine for me. It actually seems to break it if you change the viewBox size.

Reply:
If you keep the view box as it is , and change the width and height to 50px does it work fine ?

Reply:
Yes that works fine too